resource_transip_vps.go

```diff
@@ -153,24 +156,24 @@ func resourceVpsCreate(d *schema.ResourceData, m interface{}) error {
 
 	}
 
-	d.SetId(name)
-
 	d.Set("install_text", InstallText)
+	setVpsID(d, m)
 
 	return resourceVpsRead(d, m)
 }
 
 func resourceVpsRead(d *schema.ResourceData, m interface{}) error {
-	name := d.Id()
+	name := d.Get("name").(string)
```

Comment:
Can you explain a little better why this change is needed?

Reply:
It had something to do with the consistency of the unique ID TransIP has for your VPS and the description you give it in the resource. But reading it now, I think the inconsistency was in my head ;). Reverted it.